src/buttons/FlatButton.tsx

```diff
@@ -10,12 +10,12 @@ const TransparentButton = styled.button`
   cursor: pointer;
 `;
 
-const FlatButton = ({ children }) => {
+const FlatButton: React.FC = ({ children }) => {
```

Comment:
I think React.FC is deprecated

Reply:
``React.SFC`` is deprecated instead of ``React.FC``

Refer to: 
https://twitter.com/dan_abramov/status/1057795471203287040?s=20
https://stackoverflow.com/questions/53885993/react-16-7-react-sfc-is-now-deprecated